src/components/Categories.js

```diff
@@ -1,4 +1,6 @@
+import React from 'react';
+
 const Categories = () => <p>Under construction</p>;
-  <button type="button">Check Status</button>;
+  <button type="button">Check Progress</button>;
```

Comment:
I noticed that you put the `store` in your `app.js`. I think it would be okay to wrap the `main React component` in Provider and assign store according to the project requirement.

```
    import store from './redux/configureStore';

    ReactDOM.render(
    <Provider store={store}>
        <App />
    </Provider>,
    document.getElementById('root')
    )
```

Reply:
You are 100% correct. Just fixed this one. Thank you.